container-start.sh

```diff
@@ -20,4 +20,4 @@ mkdir -p customize
   && sed -i "s/logToStdout: .*/logToStdout: ${LOG_TO_STDOUT},/g" customize/config.js
 
 
-exec node ./server.js
\ No newline at end of file
+exec node --use_strict ./server.js
```

Comment:
I'm not sure what version of nodejs ships with Ubuntu these days, but when developing CryptPad we use [NVM](https://github.com/creationix/nvm). Personally I'm using node-v.6.6.0.

If this works, there's nothing to correct, but I'll be able to guarantee that the version I use will continue to work.